cart.js

```diff
@@ -0,0 +1,120 @@
+let products=[{  
+    id:1,
+    name: "Perfume",
+    price: "$100",
+    category:"Cosmetics",
+    imageURL:"images/perfume.jpg" ,
+        },{
+    id:2,
+    name: "Lipstick",
+    price: "$25",
+    category:"Cosmetics",
+    imageURL:"images/lipstick.jpg" ,
+        },{
+    id:3,
+    name: "Apple Airpods",
+    price: "$100",
+    category:"Technology",
+    imageURL:"images/airpods.jpg" ,
+        }];
+
+let cards = document.createElement("div");
+let cardsContainer = document.getElementsByClassName("cards-container")[0];
+cardsContainer.appendChild(cards);
+
+products.map(function(product){
+    let card= document.createElement("div");
+    let cardImage= document.createElement("img");    
+    let firstRowContainer = document.createElement("div");
+    let secondRowContainer = document.createElement("div");
+    let quantityContainer = document.createElement("div");
+    let cardName= document.createElement("h3");
+    let cardPrice = document.createElement("span");
+    let cardCategory = document.createElement("span");
+    let deleteBtn = document.createElement("button");
+    let quantity = document.createElement("input");
+    let quantityText = document.createElement("span");
+    let totalPrice= document.getElementsByClassName("total-price")[0];
+
+    cardImage.src=product.imageURL;
+    cardName.innerHTML=product.name;
+    cardPrice.innerHTML=product.price;
+    cardCategory.innerHTML=product.category;
```

Comment:
Instead of `innerHTML`, you could set `textContent` instead (for example `cardName.textContent = product.name`.  This is a little bit safer, because then unexpected characters in the product name or category can't break the page layout, since the name and category won't be added as raw html.

For example, if the product name was `Mens shoes <3` then using innerHTML could be bad because of the browser trying to interpret the `<` character as part of a HTML tag.